crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt

```diff
@@ -5,25 +5,29 @@ import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Header
 import retrofit2.http.Path
+import retrofit2.http.Query
 
 internal interface CrowdinDistributionApi {
 
     @GET("/{distributionHash}/content{filePath}")
     fun getResourceFile(
         @Header("if-none-match") eTag: String,
         @Path("distributionHash") distributionHash: String,
-        @Path("filePath") filePath: String
+        @Path("filePath") filePath: String,
+        @Query("timestamp") timeStamp: String
     ): Call<ResponseBody>
 
     @GET("/{distributionHash}/manifest.json")
     fun getResourceManifest(
-        @Path("distributionHash") distributionHash: String
+        @Path("distributionHash") distributionHash: String,
+        @Query("timestamp") timeStamp: String
```

Comment:
Not needed for manifest file, there is no caching here

Reply:
updated